route_following_plugin/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+
+<!--  
+ Copyright (C) 2018-2020 LEIDOS.
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+-->
+
+<package format="2">
+  <name>route_following_plugin</name>
+  <version>3.2.0</version>
+  <description>The strategic plugin generates maneuvers to make the vehicle follow the route. </description>
+  <maintainer email="carma@todo.todo">carma</maintainer>
```

Comment:
I'd update both of these emails to `carma@dot.gov` now that we have that email operational.

Reply:
Sure.